tubular/scripts/frontend_utils.py

```diff
@@ -127,6 +153,20 @@ def create_version_file(self):
         except IOError:
             self.FAIL(1, 'Could not write to version file for app {}.'.format(self.app_name))
 
+    def copy_js_config_file_to_app_root(self, app_config, app_name):
+        """ Creates a copy of env.config.js(x) file to the app root """
+        source = app_config.get('JS_CONFIG_FILEPATH')
+        filename = app_config.get('LOCAL_CONFIG_FILENAME', "env.config.js")
```

Comment:
[sanity check] Will each individual MFEs be able to override the default `LOCAL_JS_CONFIG_FILENAME` of `env.config.js`, as needed?

For example, if an MFE relies on an `env.stage.config.jsx` file, the copied/replicated `env.config` file in the root of the MFE should also use the `.jsx` file extension. There would be a similar case if MFEs opted to use `.ts` or `.tsx` (TypeScript) extensions as well.